param/__init__.py

```diff
@@ -27,7 +27,7 @@
 
 from .parameterized import logging_level # pyflakes:ignore (needed for eval)
 from .parameterized import shared_parameters # pyflakes:ignore (needed for eval)
-
+from .io import *
```

Comment:
Yes, it's a logical separation of concerns to put IO in a separate file, but param is a bit of a special case, because it's a library that we expect at least some users to include directly into their repo, and I think having a small total number of files will help that.  If there were lots of independent chunks that could be mixed and matched, having separate files would help that, but I don't think there are, so making IO separate would just make things less obvious for users who want to do that.